app/services/data_api/tad_subjects_export.rb

```diff
@@ -19,10 +19,66 @@ def self.latest
     end
 
     def data_for_export(*)
+      counts
       {}
     end
 
+    def counts
+      @counts ||= ActiveRecord::Base
+        .connection
+        .execute(query)
+        .to_a
+    end
+
   private
 
+    def query
+      <<-SQL
+        SELECT
+          application_forms.id,
+          ARRAY_AGG(ARRAY[application_choices.status, subjects.name]),
+          CASE
+            WHEN first_nationality IS NULL OR first_nationality = '' THEN NULL
+            WHEN array[#{uk_nationalities}]::varchar[] && array[first_nationality, second_nationality, third_nationality, fourth_nationality] THEN 'UK'
+            WHEN array[#{eu_nationalities}]::varchar[] && array[first_nationality, second_nationality, third_nationality, fourth_nationality] THEN 'EU'
+            ELSE 'Not EU'
+          END nationality,
+          CASE
+            WHEN application_forms.country IS NULL OR application_forms.country = '' THEN NULL
+            WHEN array[#{uk_country_codes}]::varchar[] && array[application_forms.country] THEN 'UK'
+            WHEN array[#{eu_country_codes}]::varchar[] && array[application_forms.country] THEN 'EU'
+            ELSE 'Not EU'
+          END domocile
+        FROM application_choices
+        INNER JOIN application_forms ON application_choices.application_form_id = application_forms.id
+        INNER JOIN course_options ON application_choices.course_option_id = course_options.id
+        INNER JOIN courses ON courses.id = course_options.course_id
+        INNER JOIN course_subjects ON courses.id = course_subjects.course_id
+        INNER JOIN subjects ON subjects.id = course_subjects.subject_id
+        WHERE application_forms.recruitment_cycle_year = 2022
```

Comment:
should probs be 
`
 WHERE application_forms.recruitment_cycle_year = #{RecruitmentCycle.current_year}
`
so it's future proof